modules/test/playwright/tests/frontend-data-set-web/customized.spec.ts

```diff
@@ -41,6 +41,36 @@ test.beforeEach(async ({fdsSamplePage, page, site}) => {
 	).toBeVisible();
 });
 
+test(
+	'Check filter is preloaded when entering on an FDS page for first time',
```

Comment:
[non-blocking] For performance, could we maybe merge this with some other tests? Merged sest could have a more generic name, something like `Check initial state of FDS features`

Reply:
Good idea, I'll look into it